### PR TITLE
Local contrast moved from exposure to detail tab

### PR DESCRIPTION
**Pull request by agriggio**

### CODE REVIEW

File: rtgui/partialpastedlg.cc [hunk 1]
```diff
@@ -313,6 +312,7 @@ PartialPasteDlg::PartialPasteDlg (const Glib::ustring &title, Gtk::Window* paren
 
     // Detail Settings:
     sharpenConn     = sharpen->signal_toggled().connect (sigc::bind (sigc::mem_fun(*detail, &Gtk::CheckButton::set_inconsistent), true));
+    localcontrastConn = localcontrast->signal_toggled().connect (sigc::bind (sigc::mem_fun(*basic, &Gtk::CheckButton::set_inconsistent), true));
```
Comment (Beep6581) [inline rtgui/partialpastedlg.cc]: @agriggio 
s/basic/detail/

Comment (agriggio) [reply rtgui/partialpastedlg.cc]: :+1: thanks!

File: rtgui/paramsedited.cc [hunk 2]
```diff
@@ -1403,18 +1403,19 @@ void ParamsEdited::combine (rtengine::procparams::ProcParams& toEdit, const rten
...
-        toEdit.localContrast.amount = mods.localContrast.amount;
-    }
-    if (localContrast.darkness) {
-        toEdit.localContrast.darkness = mods.localContrast.darkness;
-    }
-    if (localContrast.lightness) {
-        toEdit.localContrast.lightness = mods.localContrast.lightness;
-    }
+
+#define ADDSETVAL_(v, i)                                                   \
```
Comment (Beep6581) [inline rtgui/paramsedited.cc]: Are the trailing backslashes `\` needed?

Comment (agriggio) [reply rtgui/paramsedited.cc]: of course they are

**Comment by Beep6581** [comment]
Tested, no issues found.

**Comment by Beep6581** [comment]
Wouldn't it be nicer to use the formatting from 1 or 2 instead?
screenshot_20180201_123135

**Comment by agriggio** [comment]
Yes, good idea. can you take care of that please?

**Comment by Beep6581** [comment]
@agriggio sure, which style, 1 or 2?

**Comment by agriggio** [comment]
you choose

**Comment by Beep6581** [comment]
Done, +1 for merge.

**Pull request merged**